WooCommerce/Classes/POS/Presentation/Reusable Views/POSItemImageView.swift

```diff
@@ -7,19 +7,33 @@ struct POSItemImageView: View {
     let scale: CGFloat
 
     var body: some View {
-        if let imageSource = imageSource {
+        if let imageSource {
             ProductImageThumbnail(productImageURL: URL(string: imageSource),
                                   productImageSize: imageSize,
                                   scale: scale,
-                                  foregroundColor: .clear,
+                                  foregroundColor: Constants.placeholderIconColor,
                                   cachesOriginalImage: true)
         } else {
             Rectangle()
-                .foregroundColor(Color(.secondarySystemFill))
+                .foregroundColor(Constants.placeholderBackgroundColor)
+                .overlay {
+                    Image(systemName: "archivebox")
+                        .resizable()
+                        .frame(width: Constants.placeholderIconDimension, height: Constants.placeholderIconDimension)
```

Comment:
It probably wasn't written anywhere but we agreed using font sizes rather than constant sizes for images:

<img width="301" alt="image" src="https://github.com/user-attachments/assets/90f075be-dbba-44d4-9fd2-8e8bee7ed6cc" />

Plus, the icon looks smaller in the designs, but maybe setting the correct font would be solution 👍

Reply:
Good point, I updated the placeholder to be `Text` that uses the symbol font style in https://github.com/woocommerce/woocommerce-ios/pull/15089/commits/872932683824b2d7d0221e67f8ed37341e9da966 (font styles will be updated in a separate PR as well).

Reply:
Oh! I didn't know you could do `Text(Image(...))` and apply font sizes, that's neat 💯